src/main/java/uk/co/onsdigital/discovery/model/DimensionValue.java

```diff
@@ -6,15 +6,27 @@
 
 @Entity
 @Table(name = "dimension_value", uniqueConstraints = @UniqueConstraint(columnNames={"dimensional_data_set_id", "name", "value"}))
+@NamedQueries({
+        @NamedQuery(name = DimensionValue.FIND_QUERY, query = "SELECT dim FROM DimensionValue dim WHERE dim.dimensionalDataSetId = :ddsId AND dim.name = :name AND dim.value = :value")
+})
 public class DimensionValue {
 
     private static final long serialVersionUID = 1L;
 
+    // Named query to find a DimensionValue by dataset id, name and value.
```

Comment:
Can you make all of these one-line comments be actual Javadoc so that they show up in IntelliJ etc.